### DISCUSSION TITLE
How I enable the dashboard?

### DISCUSSION THREAD

**Discussion by ghost** [NvChad/NvChad#3104] [Q&A] [answered]
I've seen the Dashboard on the showcases, but idk how to enable it.
The links are broken.

**Comment by fmway** [comment]
open your chadrc config (`~/.config/nvim/lua/chadrc.lua`) and just add `nvdash.load_on_startup = true` like this:
```lua
---@type ChadrcConfig
local M = {}
-- your another config
...
M.nvdash.load_on_startup = true -- add this
...
return M
```
If you have lsp enabled, you can see all of chadrc's options:
![image](https://github.com/user-attachments/assets/27264fae-0651-4385-8b4a-c45698d8fe9f)

**Reply by ghost** [reply]
Still does not work

**Reply by siduck** [reply]
@theactualmariluski  just add M.nvdash and add the field in it? whats so hard about it... please check the default list of options and i hope you have read nvchad.com and :h nvui

**Reply by RyuZinOh** [reply]
> Still does not work
![{96505AA3-4478-4645-8961-4678C309B99D}](https://github.com/user-attachments/assets/3de9a01e-5ffe-447b-8b36-ce3ecda03640)

**Reply by Muriilow** [reply]
> @theactualmariluski just add M.nvdash and add the field in it? whats so hard about it... please check the default list of options and i hope you have read nvchad.com and :h nvui

i'm too dumb to read docs, that's what makes so hard to understand

**Reply by siduck** [reply]
@Muriilow or you're too dumb to read. Please read :h nvui. Nvchad isnt for beginners fyi

**Reply by siduck** [reply]
@RyuZinOh  dont tell me you are using v2.5 of ui repo. it works fine here

https://github.com/user-attachments/assets/bb9ed3d4-5819-4de5-bcf7-34767d19f5dc

**Reply by Muriilow** [reply]
> @Muriilow or you're too dumb to read. Please read :h nvui. Nvchad isnt for beginners fyi

setting up my own theme in nvchad was the most painful and miserable process of my entire setup too far, you have a whole website to explain easily with images and details, but somehow prefer to use nvim to do it... 

and even within neovim your documentation is very poor,  in the nvui.base46 local themes you say: 

> - Put the theme file in lua/themes folder

Let me show how someone who understands the basic of communication would explain: 

> In order to create your own setup you need to create a file in the  *~/.local/share/nvim/lazy/base46/lua/base46/themes* folder
> It's necessary to follow some syntaxes, use this arquive as a placeholder. 

Can you see the difference between a good explanation over yours ? The difference is that mine didnt expect that the user would understand before hand where was the directory. Sometimes people doesn't want to read all the UI documentation, they just want to read about *that* specific thing. And **most** of the times, that specific thing can be explained easily with some additional lines in your doccumentation. 

I'm also 100% sure that half of the people that ask questions about nvchad on forums had read your documentation before.
 
> Nvchad isnt for beginners fyi

The whole idea of this type of tool is to allow begginers to setup and start using neovim with the powers of an IDE, veterans could do that for themselves. You not only don't even know who is the main audience for your project, but you act like a dick to a lot of them who just are learning and trying to enter this linux environment.   

In the end you created this tool with your own rules and then expect everyone else to understand how to edit it using only two lines of explanation hidden inside a custom made command on neovim.  

I really hope that your take something from this text, and stop acting like the average arch linux user replying to everyone that have a simple question to read your bad documentation...

**Reply by siduck** [reply]
> setting up my own theme in nvchad

only base46 is supported

**Reply by siduck** [reply]
I have not seen meanest comment from any user on all of nvchad repos but you. Seems like you felt it hard understanding the docs. Again i said nvchad isnt for newbies. its not like other distros .


![image](https://github.com/user-attachments/assets/2c946f9a-4cf7-452c-a403-63466122d052)

> Let me show how someone who understands the basic of communication would explain:

> In order to create your own setup you need to create a file in the ~/.local/share/nvim/lazy/base46/lua/base46/themes folder
It's necessary to follow some syntaxes, use this arquive as a placeholder.

its common knowledge that the lua dir is in the config of the user? why would i tell to edit plugin code bruh?

**Reply by Muriilow** [reply]
> its common knowledge that the lua dir is in the config of the user?

Because you still don't understand that 90% of people who uses this type of tools are newbies who are starting to use vim. You can put this in the documentation but what's going to happen is that someone is going to watch mental outlaw video and download with a few commands your repository anyway. Anyone who understand how the file systems works on neovim understand how Lazy works and is probably going to have their own setup without the need for nvchad or lazyVim for example. 

> I have not seen meanest comment from any user on all of nvchad repos but you. 

I literally just said the truth here, like or not, I even did an entire text explaining how to make it better, that you ignored anyway... 

nvchad is a gateway for many people to ricing and linux, if you don't like to answering this questions just ignore and wait for someone else to respond. The smart guys, those who already understand how the linux environment works are not here, trust me. 

If you don't care and is still willing to do things your way you will continue receiving the same simple questions in your videos and repository.

**Reply by RyuZinOh** [reply]
> @RyuZinOh dont tell me you are using v2.5 of ui repo. it works fine here
> 
>  simplescreenrecorder-2025-01-16_08.58.55.mp4

well, if that helps ghost, why not

**Reply by amiv1** [reply, marked as answer]
I don't know why, but that didn't work for me. I had to add this instead:
```lua
M.nvdash = {
  load_on_startup = true
}
```

**Reply by siduck** [reply]
@gim- because its disabled by default and yes M.nvdash.load_on_startup  directly wont work

cuz nvdash field doesnt exist by default then how come one's accessing it's nested field. my mistake for marking it as answer sorry!
